samples/fabric/fabric-cli/src/data/event_sub_sample.json

```diff
@@ -8,16 +8,35 @@
             "transactionFunc": "create"
         },
         "event_publication_spec": {
-            "ccArgs": [],
+            "ccArgs": ["keyType", ""],
             "chaincodeId": "simplestate",
             "driverId": "network1",
             "ccFunc": "Create",
             "channelId": "mychannel",
-            "replaceArgIndex": "0"
+            "replaceArgIndex": 1
```

Comment:
How and where is this property used?

Reply:
This is used when destination driver submits a transaction. 
I remember this was modified in previous PR of Krishna, I don't know why the main didn't had it yet.

Reply:
The index here need to be an integer.